magpie-aws/src/main/java/io/openraven/magpie/plugins/aws/discovery/AWSResource.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2021 Open Raven Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.openraven.magpie.plugins.aws.discovery;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public class AWSResource {
+  public String arn;
+  public String resourceName;
+  public String resourceId;
+  public String resourceType;
+  public String awsRegion;
+  public String awsAccountId;
+  public String createdIso;
+  public String updatedIso;
+  public String discoverySessionId;
+  public Long maxSizeInBytes = null;
+  public Long sizeInBytes = null;
+
+  public JsonNode configuration;
+  public JsonNode supplementaryConfiguration;
+  public JsonNode tags;
+  public JsonNode discoveryMeta;
+
+  private AWSResource() {}
```

Comment:
I don't see a need for this constructor?

Reply:
Yes we don't, I just set it to private so isn't possible to use it withing the code